lib/ansible/modules/cloud/amazon/ec2_group.py

```diff
@@ -284,6 +295,27 @@ def get_target_from_rule(module, ec2, rule, name, group, groups, vpc_id):
     return group_id, ip, target_group_created
 
 
+def ensure_tags(ec2, group, tags, changed, module):
+    try:
+        resource_id = group.id
+        cur_tags = group.tags
+        if cur_tags == tags:
```

Comment:
Would it be better to silently to_text values, or simply fail if they're not a string? If someone wants 'TRUE' and it gets converted to 'True', might it be better to fail with a message suggesting they quote the value in the YAML, as opposed to silently converting it? Other things like floats could be similarly impacted.